Unable to build using DDS v2.4

If I try to build FairMQ with today DDS master, it fails with:

```
DEBUG:O2:FairMQ:0: CMake Error at fairmq/plugins/DDS/CMakeLists.txt:10 (add_library):
DEBUG:O2:FairMQ:0:   Target "FairMQPlugin_dds" links to target "DDS::dds-user-defaults" but the
DEBUG:O2:FairMQ:0:   target was not found.  Perhaps a find_package() call is missing for an
DEBUG:O2:FairMQ:0:   IMPORTED target, or an ALIAS target is missing?
```

I am using FairMQ 1.4.2. Which version will support DDS 2.4?

--- a/fairmq/plugins/DDS/CMakeLists.txt
+++ b/fairmq/plugins/DDS/CMakeLists.txt
@@ -8,13 +8,13 @@
 
 set(plugin FairMQPlugin_dds)
 add_library(${plugin} SHARED ${CMAKE_CURRENT_SOURCE_DIR}/DDS.cxx ${CMAKE_CURRENT_SOURCE_DIR}/DDS.h)
-target_link_libraries(${plugin} FairMQ DDS::dds_intercom_lib DDS::dds_protocol_lib DDS::dds-user-defaults)
+target_link_libraries(${plugin} FairMQ DDS::dds_intercom_lib DDS::dds_protocol_lib)
 target_include_directories(${plugin} PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 set_target_properties(${plugin} PROPERTIES CXX_VISIBILITY_PRESET hidden)
 
 set(exe fairmq-dds-command-ui)
 add_executable(${exe} ${CMAKE_CURRENT_SOURCE_DIR}/runDDSCommandUI.cxx)
-target_link_libraries(${exe} FairMQ DDS::dds_intercom_lib DDS::dds_protocol_lib DDS::dds-user-defaults)
+target_link_libraries(${exe} FairMQ DDS::dds_intercom_lib DDS::dds_protocol_lib)
 target_include_directories(${exe} PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 
 install(TARGETS ${plugin} ${exe}

